manifests/o/octobuild/octobuild/0.4.3/octobuild.octobuild.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: octobuild.octobuild
+PackageVersion: 0.4.3
+PackageLocale: en-US
+Publisher: Artem V. Navrotskiy

```

```suggestion
Publisher: Artem V. Navrotskiy; Marat Radchenko
```
This should match Control Panel